fastlane/Fastfile

```diff
@@ -0,0 +1,86 @@
+system("cd .. && npm install shelljs@0.7.0")
+system("cd .. && ./install.sh")
+
+lane :All do
+gradle(
+task:":libs:SalesforceAnalytics:connectedAndroidTest :libs:SalesforceSDK:connectedAndroidTest :libs:SmartStore:connectedAndroidTest :libs:SmartSync:connectedAndroidTest :libs:SalesforceHybrid:connectedAndroidTest :libs:SalesforceReact:assembleDebug :native:NativeSampleApps:RestExplorer:connectedAndroidTest",
+flags: "--continue --no-daemon --profile --stacktrace"
+)
+end
+
+lane :PR do
+gradle(
+task:":libs:SalesforceAnalytics:connectedAndroidTest :libs:SmartStore:connectedAndroidTest :libs:SmartSync:connectedAndroidTest :libs:SalesforceHybrid:connectedAndroidTest :libs:SalesforceReact:assembleDebug",
```

Comment:
@helenren It's okay if the one test fails. I think we should still add `SalesforceSDK` in here - at least we'll know if more tests fail.